src/components/study-pane.js

```diff
@@ -264,6 +264,16 @@ const StudyPane = (props) => {
 
     const [studyNotFound, setStudyNotFound] = useState(false);
 
+    const [networkLoadingFail, setNetworkLoadingFail] = useState(false);
```

Comment:
why not have an enumerate of `errorTypes` somewhere which consist of , 
```
{ type: STUDY_NOT_FOUND, errorMsgId: 'study_not_found'}
{ type: NETWORK_LOADING_FAILURE, errorMsgId: 'network_loading_failure'}
{ type: GEODATA_LOADING_FAILURE, errorMsgId: 'geodata_loading_failure'}
{ type: NONE, errorMsgId: ''}
...
```
Then you add only one state in here which keep track of the last error :
`const [error, setError] = useState(errorTypes.NONE);`
and use `error.type` or `error.errorMsgId` where needed.
It is then easier to add a new error type, without adding a new state, and without states conditional mess.